pulp_2to3_migration/app/pre_migration.py

```diff
@@ -144,9 +144,7 @@ def pre_migrate_content(content_model, mutable_type, lazy_type, premigrate_hook)
                 # it has to be updated here and not later, in case all items were migrated before
                 # and no new content will be saved.
                 pulp2content_pb.total -= 1
-                pulp2content_pb.save()
                 pulp2detail_pb.total -= 1
-                pulp2detail_pb.save()
```

Comment:
I think it's possible to never get below this `continue`, or what if it's the last iteration.

Reply:
Addressed w/ new push